### PR TITLE
Added four cards

### PR DESCRIPTION
**Pull request by gabrielazanova**

### CODE REVIEW

File: diabetes-dashboard/src/views/Dashboard.vue [hunk 1]
```diff
@@ -3,6 +3,92 @@
         <Navbar class="header"></Navbar>
         <div class="clearfix"></div>
         <div class="main">
+            <v-row>
+                <v-col cols="12" sm="6" md="6" lg="6">
+                    <p>23 May 2021</p>
+                </v-col>
+                <v-col cols="12" sm="6" md="6" lg="6" style="textAlign: right">
```
Comment (nbalasovs) [inline diabetes-dashboard/src/views/Dashboard.vue:10]: Style goes to the style section

**Review by nbalasovs — COMMENTED** [review]
Overall looks good!

File: diabetes-dashboard/src/views/Dashboard.vue [hunk 2]
```diff
@@ -178,11 +265,16 @@ export default {
...
+  /* #f2f2f2; */
   padding: 0 2% 0 2%;
 }
 .clearfix {
   height: 3vh;
-  background-color: #f2f2f2;
+  background-color: #F4FAFD;
+  /* #f2f2f2; */
+}
+p {
```
Comment (nbalasovs) [inline diabetes-dashboard/src/views/Dashboard.vue:277]: That is too global, which means that each p tag in the file will inherit these styles

File: diabetes-dashboard/src/views/Dashboard.vue [hunk 3]
```diff
@@ -3,6 +3,94 @@
...
+                                    </p>
+                                </v-col>
+                                <v-col class="rightAligned">
+                                    <p id="inlined" class="cardTime">t: 11:30</p>
+                                </v-col>
+                            </v-row>
+                        </v-card-text>
+                    </v-card>
+                </v-col>
+            </v-row>
```
Comment (nbalasovs) [inline diabetes-dashboard/src/views/Dashboard.vue:93]: Can you please move all of the related code into `Cards.vue` component?

**Review by nbalasovs — APPROVED** [review]
Looks great! That is some good quality code. Keep it up!
P.S. I'll merge that together with eCharts PR this way it will have less merge conflicts.

**Review by JfmGijsbers — APPROVED** [review]

**Pull request merged**